pages/views/blogs/ESL v4.9.0.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: content
+name: ESL v4.9.4
+title: ESL v4.9.4
+tags: [news]
+date: 2023-07-07
+link: https://github.com/exadel-inc/esl/releases/tag/v4.9.4
+---
+
+In `4.9.4` release we have the following updates: 
+ - ESL Mixin Element initialization process performance was significantly improved (33% faster initialization)
+ - ESL Animate module was moved to the stable state
+ - ESL Select module reworked using ESL Popup
+ - Print version support for components and ESL Footnotes anchor-based rendering
+ - And [more small improvements and bug fixes ...](https://github.com/exadel-inc/esl/releases/tag/v4.9.0)
```

Comment:
mb can we open this link in a new window?

Reply:
Thanks, nice catch!
PS: Issue TBD.